NorthstarDLL/scripts/client/scriptmodmenu.cpp

```diff
@@ -49,7 +50,7 @@ ADD_SQFUNC("void", NSSetModEnabled, "string modName, bool enabled", "", ScriptCo
 	return SQRESULT_NULL;
 }
 
-ADD_SQFUNC("string", NSGetModDescriptionByModName, "string modName", "", ScriptContext::SERVER | ScriptContext::CLIENT | ScriptContext::UI)
+ADD_SQFUNC("string ornull", NSGetModDescriptionByModName, "string modName", "", ScriptContext::SERVER | ScriptContext::CLIENT | ScriptContext::UI)
```

Comment:
Can you remind me why all mod methods can now return `null`?

Reply:
The way the functions are currently implemented, the return a value **only** if the mod name is found. If the mod name is not found, they return `null`. Whoever wrote those simply forgot that types like string etc. can't be `null`.